cla-backend-go/cla_manager_add/handlers.go

```diff
@@ -0,0 +1,65 @@
+// Copyright The Linux Foundation and each contributor to CommunityBridge.
+// SPDX-License-Identifier: MIT
+
+package cla_manager_add
```

Comment:
Looking back, I think we can rename the existing package of cla_manager_requests to a more generic 'cla_manager' package to be inclusive of requests, approvals, deny and your new code which allows us to add and remove CLA managers.  Let's leave this as it is and I will consolidate.

Reply:
Cool